pkg/webhook/uniteddeployment/validating/uniteddeployment_validation.go

```diff
@@ -277,11 +295,16 @@ func validateAdvancedStatefulSet(statefulSet *appsv1alpha1.AdvancedStatefulSetTe
 	if statefulSet.Spec.Replicas != nil {
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("spec", "replicas"), *statefulSet.Spec.Replicas, "replicas in advancedStatefulSetTemplate will not be used"))
 	}
-	if statefulSet.Spec.UpdateStrategy.Type == appsv1.RollingUpdateStatefulSetStrategyType &&
-		statefulSet.Spec.UpdateStrategy.RollingUpdate != nil &&
-		statefulSet.Spec.UpdateStrategy.RollingUpdate.Partition != nil {
-		allErrs = append(allErrs, field.Invalid(fldPath.Child("spec", "updateStrategy", "rollingUpdate", "partition"), *statefulSet.Spec.UpdateStrategy.RollingUpdate.Partition, "partition in advancedStatefulSetTemplate will not be used"))
+
```

Comment:
Why is the original code removed?

Reply:
Hmm, might have been a merge error on my part, will fix it thanks